src/ucp/wireup/select.c

```diff
@@ -1006,7 +1006,8 @@ static double ucp_wireup_aux_score_func(const ucp_worker_iface_t *wiface,
 }
 
 static void ucp_wireup_fill_aux_criteria(ucp_wireup_criteria_t *criteria,
-                                         unsigned ep_init_flags)
+                                         unsigned ep_init_flags,
+                                         unsigned prefer_cb_async_cap)
```

Comment:
pass mandatory_flags as parameter, instead of prefer_cb_async_cap flag

Reply:
Done. The ```mandatory_flags``` is either ```1```(select AUX transport with async active message callback) or ```0```